base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -73,11 +76,13 @@ public VideoStreamingParameters(){
     }
 
     public VideoStreamingParameters(int displayDensity, int frameRate, int bitrate, int interval,
-                                    ImageResolution resolution, VideoStreamingFormat format){
+                                    double scale, ImageResolution resolution,
```

Comment:
Please deprecate the old constructor and add a new one with the new signature.

Reply:
DONE